app/views/conference_registrations/show.pdf.prawn

```diff
@@ -0,0 +1,148 @@
+prawn_document(filename: @pdf_filename, page_layout: :landscape, :page_size => [630.28,721.89]) do |pdf|
+	
+	# SAMPLE 1
```

Comment:
Please remove the code from the examples we don't want to use and leave only the final one.